promgen/static/js/promgen.js

```diff
@@ -31,6 +31,22 @@ function update_page(data) {
   }
 }
 
+// Wait until DOM is loaded to add our extra listeners
+document.addEventListener("DOMContentLoaded", function () {
+  /*
+    Copy to shortcut
+
+    Example: <code data-copyto="#id_duration">30s</code>
+  */
+  document.querySelectorAll("[data-copyto]").forEach(srcElement => {
+    srcElement.style.cursor = "pointer";
+    srcElement.addEventListener("click", e => {
+      var dstElement = document.querySelector(srcElement.dataset.copyto);
```

Comment:
I always recommend using `const` by default, unless the variable needs to be reassigned, in which case I recommend using `let`. In my opinion `var` should always be avoided.

That is my opinion on JS best practices, but you may have others, hence this comment is optional.

Reply:
Sounds reasonable. I think I just don't have enough awareness of javascript best practices since I don't write it as much. Changed to const but just force-pushed for the single, small change.

Reply:
Re-approved :+1: